terraform/import_branch_protection.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/bash
+repos=("discord" "ffmpeg-sdk" "ci-screenshots" ".github" "mattermost-desktop" "signal-desktop" "ci" "ghvmctl" "sublime-text" "webkitgtk-sdk" "jenkins" "eclipse" "sentry" "obs-studio" "tmnationsforever" "scummvm" "xonotic" "atom" "offlineimap" "sommelier-core" "android-studio" "ffmpeg" "opentyrian" "spelunky" "flightgear" "sdlpop" "mumble" "corsixth" "magic-wormhole" "alacritty" "gimp" "helm" "sublime-merge" "minetest" "opentoonz" "fork-and-rename-me" "axel" "dosbox-x" "ddgr" "term2048" "arduino" "photoscape" "sommelier" "gradle" "s4a" "cumulonimbus" "android-studio-canary" "cassandra" "cryptool" "thelounge" "irssi" "marktext" "fkill" "pyradio" "mdl" "mosaic" "pogo" "unifi" "nano" "opentoonz-morevna" "kompozer" "get-iplayer" "simplenote" "mutt" "inadyn" "duckmarines" "brackets" "wordpress-desktop" "mrrescue" "wire" "tcpie" "slack-term" "links" "wethr" "qtctl" "bridge-designer" "climate-trail" "hello-snap-travis-ci")
```

Comment:
I'd prefer to see this array declared with one repo per line, only because when we commit changes the diffs will be much easier to grok :)